src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -119,6 +119,14 @@ public boolean isUserRegistered(String username) {
     return false;
   }
 
+  /** Return true if the given username is known as an admin user to the application. */
+  public boolean isUserAdmin(String username){
```

Comment:
Nice to extract the functionality to the store instead of having it in the servlet. @Dnae-ferguson's prototype have changes to the user model that may be useful to give an idea of a way to implement it. Do lookout for her PR :)